src/Domain/Students/MedicalConditionGateway.php

```diff
@@ -19,36 +19,28 @@
 
 namespace Gibbon\Domain\Students;
 
-use Gibbon\Domain\Traits\TableAware;
 use Gibbon\Domain\QueryCriteria;
 use Gibbon\Domain\QueryableGateway;
+use Gibbon\Domain\ScrubbableGateway;
+use Gibbon\Domain\Traits\Scrubbable;
+use Gibbon\Domain\Traits\TableAware;
+use Gibbon\Domain\Traits\ScrubByPerson;
 
 /**
  * @version v21
  * @since   v21
  */
-class MedicalConditionGateway extends QueryableGateway
+class MedicalConditionGateway extends QueryableGateway implements ScrubbableGateway
 {
     use TableAware;
+    use Scrubbable;
+    use ScrubByPerson;
 
-    private static $tableName = 'gibbonMedicalCondition';
-    private static $primaryKey = 'gibbonMedicalConditionID';
+    private static $tableName = 'gibbonPersonMedicalCondition';
+    private static $primaryKey = 'gibbonPersonMedicalConditionID';
 
     private static $searchableColumns = [];
 
-    /**
-     * @param QueryCriteria $criteria
-     * @return DataSet
-     */
-    public function queryMedicalConditions(QueryCriteria $criteria)
```

Comment:
I'm interested to know why this one was removed. Is it inherited? in some form?

Reply:
It wasn't necessarily removed, that's an artifact of the version comparison. I realized our naming conventions for the new MedicalConditionGateway overlapped with the need for a MedicalConditionGateway for individual users. So, I moved the gateway that manages the list of potential conditions to School Admin, where it's managed, so I could add the gateway for students to the Student namespace. The other option was finding and renaming all the existing MedicalGateway uses to PersonMedicalGateway. The first option seemed better, since that gateway was just introduced this version.